src/pages/index.js

```diff
@@ -20,58 +33,205 @@ const enableValidation = (settings) => {
 };
 
 //User Info
-const userinfo = new UserInfo(".profile__name", ".profile__subtitle");
+const userinfo = new UserInfo(".profile__name", ".profile__subtitle", "#profile-avatar");
 
 //Card
-function createCard(item) {
-  const cardElement = new Card(item, "#card-template", handleImageClick);
-  return cardElement.getCardElement();
+function createCard({name, link, isLiked, _id}) {
+  return new Card({name, link, isLiked, _id}, "#card-template", handleImageClick, 
+  handleTrashButtonClick, 
+  handleHeartButton,
+  ).getCardElement();
+ // const trashButton = cardElement.getTrashButton();
+ // trashButton.addEventListener("click", ()=> {
+   // popUpConfirm.open();
+ // });
+
+  //return cardElement.getCardElement();
 }
 
+
 //Section
 const section = new Section(
   {
-    items: initialCards,
+    items: [],
     renderer: (item) => {
       const cardElement = createCard(item);
       section.addItem(cardElement);
     },
   },
-  ".cards__content"
+  ".cards__content",
 );
 section.renderItems();
 
+Promise.all([api.getInitialCards(), api.getUserInfo()])
+//process the result
+.then(([cardData, formData]) => {
+ // console.log("Card Data:", cardData);
+ // console.log("Form Data:", formData)
+ userinfo.setUserInfo(formData);
+ userinfo.setAvatar(formData.avatar);
+ cardData.forEach((item) => {
+ 
+  //console.log("is like and unlike here at ForEach?",cardData)
+  //cardData.setLikeStatus(cardData.isLiked)
+  //const cardElement = createCard(item)
+   section.addItem(createCard(item));
```

Comment:
Please, use `renderItems` to render the initial cards


```js

    section.renderItems(cardData);

```

Reply:
i tried, using section.renderItems(cardData) and majority of my cards doesn't show up in my cards gallery, somehow it only works with section.addItem does this mean i have to re-do this whole part?